### PR TITLE
toLowerCase fixed

### PR DESCRIPTION
**Pull request by codeclude**

### CODE REVIEW

File: Packtpub_Book_List_Filter.user.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ $(document).ready(function () {
   $('#myCustomSearchBox').on('input', function () {
     var newValue = $(this).val().toLowerCase();
     $('.product-line').each(function () {
-      if ($(this).attr('title').toLowerCase().indexOf(newValue) > -1) {
+      if ($(this).attr('title') && $(this).attr('title').toLowerCase().indexOf(newValue) > -1) {
```
Comment (kejn) [inline Packtpub_Book_List_Filter.user.js]: Please extract `$(this).attr('title')` to a variable

**Review by kejn — CHANGES_REQUESTED** [review]
Variable requires extracting.

File: Packtpub_Book_List_Filter.user.js (hunk 1 above)
Comment (codeclude) [reply Packtpub_Book_List_Filter.user.js]: np

**Review by kejn — APPROVED** [review]
Approved. Thx!

**Comment by kejn** [comment]
Closes issue #1

**Pull request merged**